src/components/bulk-selection-snackbar/BulkSelectionSnackbar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import CloseIcon from "./CloseIcon";
+import { SBulkSelectionSnackbar } from "./styles/SBulkSelectionSnackbar";
+import { SBulkSelectionSnackbarChildren } from "./styles/SBulkSelectionSnackbarChildren";
+import { SBulkSelectionSnackbarCloseButton } from "./styles/SBulkSelectionSnackbarCloseButton";
+import { SBulkSelectionSnackbarSelected } from "./styles/SBulkSelectionSnackbarSelected";
+import { SBulkSelectionSnackbarSelectedBadge } from "./styles/SBulkSelectionSnackbarSelectedBadge";
+import { SBulkSelectionSnackbarWrapper } from "./styles/SBulkSelectionSnackbarWrapper";
+
+interface IBulkSelectionSnackbar {
+  selectedItems: any[];
```

Comment:
Why this need to be a list of elements? Why not just the count?

Reply:
I did it like this initially because I thought we'd need to return the selectedItems in some function, but this wasn't the case! I'll change it to being just the count